src/components/ApplicationTile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import Text from '../Text';
+import styles from './styles.css';
+import getFormattedTimeDifference from '~/utils/getFormattedTimeDifference';
+
+interface Props {
+  companyName: string;
+  roleName: string;
+  status: 'sent' | 'inProgress' | 'rejected' | 'accepted' | 'hired' | 'archived';
```

Comment:
Maybe Status could be separated type?

Reply:
Yes, it is but I don't know why I didn't use it here. Fixed